website/ocular-config.js

```diff
@@ -13,6 +15,12 @@ const ALIASES = {
   'nebula.gl/dist': MAIN_SRC_DIR,
   'nebula.gl': MAIN_SRC_DIR,
 
+  '@nebula.gl/core/dist': CORE_SRC_DIR,
```

Comment:
Instead of duplicating the aliases in many places, we used to have an `aliases.js` at the top that we imported in webpack-config.js and here.

Now ocular-dev-tools automatically generates the aliases (but you are using flow and jest so that won't work for you)